### DISCUSSION TITLE
Overlay public gpx

### DISCUSSION THREAD

**Discussion by drazde** [osmandapp/OsmAnd#13138] [Ideas]
Having the ability to overlay public gpx tracks on top of the map. This is usefull for found the most popular path specally  on the mountain.

PS: like Heat Map on Strava routes planner

**Comment by tommycrock** [comment]
When you say 'like Heat Map on Strava', what source do you suggest? Osmand doesn't collect user movements centrally. To get Strava heat map in osmand follow these instructions https://wiki.openstreetmap.org/wiki/Strava

**Reply by drazde** [reply]
> 
> 
> When you say 'like Heat Map on Strava', what source do you suggest? Osmand doesn't collect user movements centrally. To get Strava heat map in osmand follow these instructions https://wiki.openstreetmap.org/wiki/Strava

Yes, but Openstreetmap have many public gps and osmand could show it. 
Like the site does:
![immagine](https://user-images.githubusercontent.com/2643448/140909121-b90f3e88-6482-46c0-b489-ab479205904c.png)

In this example you coud decide if take a path or another based on popularity

**Reply by Nine-Tailed** [reply]
Osmand can show these tracks, it is enough to create a map with the parameters as in the screenshot

![Screenshot_2022-02-19-00-00-22-061_net.osmand.plus.jpg](https://user-images.githubusercontent.com/12549744/154760583-8af4bba7-8e82-4be7-97fc-c19a94ae76f1.jpg)

https://gps.tile.openstreetmap.org/lines/{0}/{1}/{2}.png
